project_event/models/task_template.py

```diff
@@ -11,6 +11,10 @@ class TaskTemplate(models.Model):
     name = fields.Char(
         string='Name',
     )
+    resource_id = fields.Many2one(
+        'resource.resource',
+        string='Resource',
+    )
```

Comment:
@rimbendhaou: At line 48, Format the line as other fields declaration. Change the string to meaning text.